pkg/infoschema/infoschema_v2.go

```diff
@@ -613,10 +613,9 @@ retry:
 			time.Sleep(200 * time.Millisecond)
 			goto retry
 		}
-		// TODO: error could happen, so do not panic!
-		panic(err)
+		return nil, errors.Trace(err)
```

Comment:
Should we retry for this kind of error?

Reply:
No.  @lance6716 
Most of the retryable errors have been handled in the tikv client layer, such as region split etc.
The very rare 'flashback region in progress' error is handled in this function already.
I can't find any errors that we can retry here.